src/librustdoc/clean/mod.rs

```diff
@@ -1160,6 +1160,10 @@ enum PathKind {
     Type,
 }
 
+fn resolution_failure(cx: &DocContext, path_str: &str) {
+    cx.sess().warn(&format!("[{}] cannot be resolved, ignoring it...", path_str));
```

Comment:
Are there spans available that we can use to report the warnings?

Reply:
It'll be part of another pr. I think this one is already big enough.